cclient/api/core/core_init.c

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
Can we just call this `core.c` ?

Reply:
it does the initialization job, not contains core methods.

Reply:
It also contains `destroy` so I  find the name misleading